src/Uno.UWP/UI/Core/CoreWindow.Pointers.Managed.cs

```diff
@@ -24,6 +24,23 @@ public partial class CoreWindow : ICoreWindowEvents
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyDown;
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyUp;
 
+#if __SKIA__
+	internal event TypedEventHandler<CoreWindow, WrappedKeyEventArgs>? NativeKeyDownReceived;
+	internal event TypedEventHandler<CoreWindow, WrappedKeyEventArgs>? NativeKeyUpReceived;
+
+	internal class WrappedKeyEventArgs
```

Comment:
Why do we actually need this wrapper? Is `KeyEventArgs` alone not enough?

Reply:
We don't. I thought KeyEventArgs was a value type for some reason. My bad.

Reply:
@ramezgerges Note: even if it was a value type, you can pass it around by reference with `ref` keyword :)